FrontEnd/src/components/ProfilePage/FormComponents/ContactsInfo.js

```diff
@@ -1,31 +1,27 @@
 import css from './FormComponents.module.css';
 import { useState, useEffect } from 'react';
-
+import { useUser, useProfile } from '../../../hooks/';
 import FullField from './FormFields/FullField';
 import HalfFormField from './FormFields/HalfFormField';
+import Loader from '../../loader/Loader';
 
 const LABELS = {
-    'phoneNumber': 'Телефон',
-    'companyEmail': 'Електронна пошта',
-    'companySite': 'Сайт',
+    'phone': 'Телефон',
     'address': 'Адрес(и)',
-    'Facebook': 'Facebook',
-    'Instagram': 'Instagram',
-    'Tiktok': 'Tiktok',
-    'LinkedIn': 'LinkedIn',
-    'Youtube': 'Youtube',
 };
 
 const ContactsInfo = (props) => {
-    const [user, setUser] = useState(props.user);
+    const { user } = useUser();
+    const { profile: mainProfile, mutate: profileMutate } = useProfile();
+    const [profile, setProfile] = useState(props.profile);
```

Comment:
Same, you can pass as a props `setProfile`

Reply:
I used [profile, setProfile] for form data, so i don`t understand a bit, why do i need pass this setter across files, when it needs only for this one. And i need to send patch request before updating user's data globally. If I didn`t understand smth, Can you explain me it another way?

Reply:
It's not a good practice to have many sources of trues. Btw, what is the difference between `props.profile` and `mainProfile`? Do you need to pass it as a props at all?

Reply:
I suppose that `mainProfile` and `props.profile` contain the same data. 
You may utilize `useEffect` hook https://react.dev/reference/react/useEffect

Reply:
I suppose that `mainProfile` and `props.profile` contain the same data. 
You may utilize `useEffect` hook https://react.dev/reference/react/useEffect
https://github.com/ita-social-projects/Forum/pull/259/files/d328d17149692bb74a95f30f9ac53fe52958acfa#r1373444803